pkg/registry/core/serviceaccount/storage/token.go

```diff
@@ -97,9 +97,13 @@ func (r *TokenREST) Create(ctx context.Context, name string, obj runtime.Object,
 	}
 
 	// Lookup service account
-	svcacctObj, err := r.svcaccts.Get(ctx, name, &metav1.GetOptions{})
+	// set ResourceVersion=0 to allow this to be read/served from the apiservers watch cache
+	svcacctObj, err := r.svcaccts.Get(ctx, name, &metav1.GetOptions{ResourceVersion: "0"})
```

Comment:
@hakuna-matatah where else is this pattern used?

Reply:
You mean cache look up ? We do that for nodes here https://github.com/kubernetes/kubernetes/blob/6fdacf04117cef54a0babd0945e8ef87d0f9461d/pkg/registry/core/serviceaccount/storage/token.go#L162